### PR TITLE
Add contributors to the pymc sprint

### PR DESCRIPTION
**Pull request by LucyJimenez**
Add five contributors to the pymc sprint.

### CODE REVIEW

File: 2022-07_sprint/sprint_parties/contributors.md [hunk 1]
```diff
@@ -259,4 +259,58 @@ Callistus Ndemo
...
 :::
+:::{grid-item}
+
+<img alt="Image for contributor template" src="../../_static/contributors/blank_person.jpg" class="rounded-circle">
+
+Andrianina David Raharijao
+
+NYC, United States
+
+New contributor
```
Comment (OriolAbril) [inline 2022-07_sprint/sprint_parties/contributors.md:270]: I am not sure this should be added inside the personal squares cc @reshamas

File: 2022-07_sprint/sprint_parties/contributors.md [hunk 2]
```diff
@@ -259,4 +259,58 @@ Callistus Ndemo
...
+
+[{fab}`github`](https://github.com/Icyshaman)
+[{fab}`linkedin-in`](https://www.linkedin.com/in/vikas-vishwakarma-04b738193/)
+:::
+:::{grid-item}
+
+Daniel Saunders
+
+[{fab}`github`](github.com/daniel-saunders-phil)
```
Comment (OriolAbril) [inline 2022-07_sprint/sprint_parties/contributors.md:313]: This link and the one below are raw urls, but they don't have the https:// start, so sphinx treats them as cross-references. This is what is breaking the CI job.

File: 2022-07_sprint/sprint_parties/contributors.md (hunk 1 above)
Comment (reshamas) [inline 2022-07_sprint/sprint_parties/contributors.md:270]:
```suggestion
```

File: 2022-07_sprint/sprint_parties/contributors.md [hunk 3]
```diff
@@ -259,4 +259,58 @@ Callistus Ndemo
...
+:::
+:::{grid-item}
+
+<img alt="Image for contributor template" src="../../_static/contributors/blank_person.jpg" class="rounded-circle">
+
+Abhilash Babu
+
+Germany
+
+New contributor
```
Comment (reshamas) [inline 2022-07_sprint/sprint_parties/contributors.md:290]:
```suggestion
```

File: 2022-07_sprint/sprint_parties/contributors.md [hunk 4]
```diff
@@ -259,4 +259,58 @@ Callistus Ndemo
...
+:::
+:::{grid-item}
+
+<img alt="Image for contributor template" src="../../_static/contributors/blank_person.jpg" class="rounded-circle">
+
+Vikas Vishwakarma
+
+Bhopal, Madhya Pradesh, India
+
+New contributor
```
Comment (reshamas) [inline 2022-07_sprint/sprint_parties/contributors.md:304]:
```suggestion
```

File: 2022-07_sprint/sprint_parties/contributors.md (hunk 1 above)
Comment (reshamas) [inline 2022-07_sprint/sprint_parties/contributors.md:270]:
```suggestion
```

Comment (reshamas) [reply 2022-07_sprint/sprint_parties/contributors.md:270]: Yes, agreed.  Will remove this.

**Comment by reshamas** [comment]
@LucyJimenez 
It looks like Daniel Saunders needs an image placeholder.

Preview:  https://pymc-data-umbrella--123.org.readthedocs.build/en/123/2022-07_sprint/sprint_parties/contributors.html

**Comment by LucyJimenez** [comment]
Ok, I'll do it. Could you please give access to the pictures?

**Comment by reshamas** [comment]
@LucyJimenez Looks like there are merge conflicts here.

**Comment by OriolAbril** [comment]
When there are multiple PRs open editing the same files, generally merging the first one generates merge conflicts on the other ones. Even if there weren't any originally. I'll fix it

**Comment by reshamas** [comment]
@OriolAbril I would like to learn how to fix it.  Can you do it on GitHub?

**Comment by OriolAbril** [comment]
> @OriolAbril I would like to learn how to fix it. Can you do it on GitHub?

Saw that too late. It can be done on GitHub. For conflicts whose resolution I see clearly (here it was keeping the content from both branches, sometimes it is combining the content on code and I need to run tests too to make sure I get it right) I click on the button that is shown below the merge conflicts message. This opens a raw text editor with the conflicts marked <, > and = signs, something like:

```
>>>>people01
content from people01 branch
====
content from main branch
<<<<main
```

I can try to do the next one over a videocall or take some screenshots. It is also possible to do it locally with similar steps.

**Pull request merged**